talk-9-js-in-practice-server/slides.md

```diff
@@ -1,29 +1,166 @@
 # Javascript in Practice: Server
-
+---
 ## Outline
-
-### Intro to Node
+- Node JS
+- Server-client interaction
+- Async calls vs. Multithreading
+- npm
+- Middleware
+- Routing/Templating
+- MongoDB/Mongoose
+- Other Server Options
+---
+## What is Node?
 - Essentially a headless V8 engine allows for Javascript outside a browser environment
-- Node as a server, Express as the most common and easy Node server
+- Written in C++, very fast/scalable
+- Node itself is not a web server
+- ExpressJS is an implementation of a web server using Node
+
+Note: Not being run in browser, no DOM. Not running on JavaScript itself, C++ running and interpretting JavaScript.
+"node" "console.log('hello');"
```

Comment:
Is this line a code example? Should it come before the Note? Is this the right syntax? (I thought it needs backticks: ``)